packages/agent/conf/examples/oid4vci_metadata/banqup.json

```diff
@@ -110,6 +110,123 @@
           "ES256K",
           "EdDSA"
         ]
+      },
+      {
+        "display": [
+          {
+            "name": "Verified Identity Credential",
+            "background_color": "#322577",
+            "logo": {
+              "url": "https://i.ibb.co/4P0C0rL/banqup-logo-white.png",
+              "alt_text": "Banqup logo"
+            }
+          },
+          {
+            "locale": "en-US",
+            "name": "Verified Identity Credential",
+            "background_color": "#322577",
+            "logo": {
+              "url": "https://i.ibb.co/4P0C0rL/banqup-logo-white.png",
+              "alt_text": "Banqup logo"
+            }
+          },
+          {
+            "locale": "nl-NL",
+            "name": "Geverifieerd Identity Credential",
+            "background_color": "#322577",
+            "logo": {
+              "url": "https://i.ibb.co/4P0C0rL/banqup-logo-white.png",
+              "alt_text": "Banqup logo"
+            }
+          }
+        ],
+        "order": [
+          "firstName",
+          "lastName",
+          "emailAddress",
+          "bsnNumber"
+        ],
+        "credentialSubject": {
+          "firstName": {
+            "value_type": "string",
+            "display": [
+              {
+                "name": "First name"
+              },
+              {
+                "name": "First name",
+                "locale": "en-US"
+              },
+              {
+                "name": "Voornaam",
+                "locale": "nl-NL"
+              }
+            ]
+          },
+          "lastName": {
+            "value_type": "string",
+            "display": [
+              {
+                "name": "Last name"
+              },
+              {
+                "name": "Last Name",
+                "locale": "en-US"
+              },
+              {
+                "name": "Achternaam",
+                "locale": "nl-NL"
+              }
+            ]
+          },
+          "emailAddress": {
+            "value_type": "string",
+            "display": [
+              {
+                "name": "Email address"
+              },
+              {
+                "name": "Email address",
+                "locale": "en-US"
+              },
+              {
+                "name": "Email adres",
+                "locale": "nl-NL"
+              }
+            ]
+          },
+          "bsnNumber": {
+            "value_type": "string",
+            "display": [
+              {
+                "name": "BSN number"
```

Comment:
bsn means, citizen service number, so now it actually says citizen service number number. :)

i'm fine with this for now, just noting it here